docs/UserGuide.md

```diff
@@ -210,6 +208,9 @@ Be very careful with how a command is formatted! If the `…` is found within th
 
 --------------------------------------------------------------------------------------------------------------------
 <div style="page-break-after: always;"></div>
+<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
+Avoid giving irrelevant inputs to commands that do not accept those inputs. For example, `mark` expects a `<person index>` input and a `debt/<debt index…>` input, but not a `t/<tag>` input. Giving such inputs may cause unexpected behaviour in PayMeLah!
```

Comment:
Instead of saying unexpected behaviour, maybe we can just say the command will not be carried out successfully?

Reply:
In some cases the command will work but unexpectedly; try `add n/jack t/friend a/123 jack st d/meal`, which considers `d/meal` part of the address input